.changeset/itchy-crabs-return.md

```diff
@@ -0,0 +1,28 @@
+---
+"@aws-amplify/ui-react-storage": minor
+"@aws-amplify/ui-react": minor
+---
```

Comment:
Did Changeset bot not detect the ui package changes? These changes should cause a bump?